configserver/src/test/java/com/yahoo/vespa/config/server/ApplicationRepositoryTest.java

```diff
@@ -117,6 +120,14 @@ public void createAndPrepareAndActivate() {
         assertTrue(result.configChangeActions().getRestartActions().isEmpty());
     }
 
+    @Test
+    public void testSuspension() {
+        deployApp(testApp);
+        assertFalse(applicationRepository.isSuspended(applicationId()));
+        orchestrator.suspend(applicationId());
+        assertTrue(applicationRepository.isSuspended(applicationId()));
```

Comment:
I'd also check if you can resume, and then see that the application is resumed.

Reply:
Well ... nevermind. It's just backed by a mock, huh?

Reply:
Yep.